apps/kyb-app/src/components/organisms/DynamicUI/StateManager/components/ActionsHandler/action-handlers/value-apply/types.ts

```diff
@@ -0,0 +1,31 @@
+import { BaseActionParams, Rule } from '@/domains/collection-flow';
+
+export type ValueApplySelectorType = 'raw' | 'json-logic' | 'pick';
+
+export interface RawSelector<TValue = unknown> {
+  type: 'raw';
+  value: TValue;
+}
+
+export interface PickSelector {
+  type: 'pick';
+  value: string;
+  defaultValue?: unknown;
```

Comment:
Can defaultValue be different in type than value?

Reply:
It can, but does not make sense.

Reply:
So the question is even it makes sense to use a generic shared between value and defaultValue or make defaultValue also `string`.

Reply:
Updated naming. Default value can be anything, type has to be resolved in component/function/any where it used.